app/modules/flappybird/models_flappybird.py

```diff
@@ -14,6 +14,7 @@ class FlappyBirdScore(Base):
     user_id: Mapped[str] = mapped_column(ForeignKey("core_user.id"))
     user: Mapped[CoreUser] = relationship("CoreUser", init=False)
     value: Mapped[int]
+    key: Mapped[int]
```

Comment:
Why are we storing the key in the database ?

Reply:
> Why are we storing the key in the database ?

That was a mistake, I think it's fixed now.